GOVERNANCE.md

```diff
@@ -360,7 +360,6 @@ If the pull request submitter is not a Core Member, the PR can be closed by any
 
 Astro features are discussed using a model called [Consensus-seeking decision-making](https://en.wikipedia.org/wiki/Consensus-seeking_decision-making). This model attempts to achieve consensus on all significant changes to Astro, but has a fallback voting procedure in place if consensus appears unattainable.
 
-**Who can vote:** All [TSC](#technical-steering-committee-tsc) members.
```

Comment:
With this line gone entirely, now I'm not sure it ever states anywhere among whom consensus is achieved? (Or I missed it?) Anyone who happens to be on the call? (And, these calls used to be public, so that would include public/maintainers equally?)

Maybe this means clarifying below in number 4, something like:

> 4. During an RFC meeting, the person leading the call will attempt to achieve consensus on the RFC proposal among all attending Astro (L??) maintainers .